### PR TITLE
PROD-3257 Versioning & README -> master

### PR DESCRIPTION
**Pull request by brooketopcoder**
This PR updates the README to include information about versioning and adds the version number to the ouput JS script.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,3 +93,26 @@ The component renders the footer for the page
...
+
+https://uni-nav.topcoder-dev.com/tc-universal-nav-#.js
+
+Each application will need to determine which version of the nav it uses by setting the URL for the script.
+
+For example, non-production environments will probably want to point to the script hosted on  `//uni-nav.topcoder-dev.com`
+
+### Minor Version upgrades
+
+Minor version updates will be rolled out silently to all clients.
```
Comment (vas3a) [inline README.md:112]: pretty cool!

File: vite.config.ts [hunk 2]
```diff
@@ -3,26 +3,28 @@ import { defineConfig } from 'vite'
...
     },
-  }
+    build: {
+        minify: true,
+        // cssCodeSplit: true,
+        lib: {
+            entry: resolve(__dirname, 'src/main.ts'),
+            formats: ['es'],
+            name: 'TcUniversalNav',
+            fileName: `tc-universal-nav-${pkg.version.split('.')[0]}`
```
Comment (vas3a) [inline vite.config.ts:27]: Just a proposal, can we name it like `tc-universal-nav.v1.js`?

**Review by vas3a — APPROVED** [review]
looks great!

**Review by testflyjets — APPROVED** [review]

File: vite.config.ts (hunk 2 above)
Comment (brooketopcoder) [reply vite.config.ts]: @vas3a per conversation, updated it to `/v1/tc-universal-nav.js`

**Pull request merged**